dim-testsuite/runtest.py

```diff
@@ -105,7 +106,7 @@ def clean_database():
     if not hasattr(clean_database, 'dumped'):
         commands.extend([
             "echo 'drop database dim; create database dim;' | " + DIM_MYSQL_COMMAND,
-            'manage_db clear',
```

Comment:
Should we leave the old manage_db and manage_dim calls here or do you feel better switching to the flask calls?

Reply:
Okay.